awot/graph/sonde.py

```diff
@@ -6,7 +6,7 @@
 from .skew import SkewXTick
 
 
-def plot_skewt_logp(data, **kwargs):
+def plot_skewt_logp(data, instance, **kwargs):
 
     '''
     method to plot sounding or dropsonde data

```

Comment:
I think we should give a bit more info, for example for L11 -19 should mimic the rest of the package:
```
    '''
    method to plot sounding or dropsonde data

    Inputs
    ------
    dictionary of sounding data

    keyword arguments min, max, titles, and label.
    '''
```

should change to something like:
```
    '''
    Plot rawinsode or dropsonde data.

    Parameters
    ---------------
    data : dict
        AWOT sounding data dictionary. For dropsondes, this is often a dictionary of dictionaries.
    instance : str
        Key for identifying dictionary to plot.

    keyword arguments min, max, titles, and label.
    '''
```